dev/test-studio/sanity.config.ts

```diff
@@ -151,6 +151,9 @@ export default defineConfig([
     plugins: [sharedSettings()],
     basePath: '/test',
     icon: SanityMonogram,
+    // unstable_serverActions: {
+    //   enabled: true,
+    // },
```

Comment:
This is commented out because there's a bit of "low editor FPS" that needs to be debugged but is kind of an annoyance at present (another good reason for getting these changes in earlier!)

This sluggishness is also, by my testing, present in #6116 , so is likely closer to the server than to the code splitting.

Reply:
How does it compare to `next`? There shouldn't be much (or any?) extra CPU-bound work being added by these changes, so don't quite understand why this happens.

Reply:
I'm unable to recreate today! 😔 But I did notice next acting faster then for some reason. If anything, this is reason to keep this enabled in test studio so we can capture, so I've uncommented it for now.